app/assets/stylesheets/_variables.scss

```diff
@@ -44,7 +44,7 @@ $link-hover-decoration: underline !default;
 //## Font, line-height, and color for body text, headings, and more.
 
 $font-family-sans-serif:  "Helvetica Neue", Helvetica, Arial, 'Apple Symbols','Menlo','Arial Unicode MS' sans-serif !default; //'Apple Symbols','Menlo','Arial Unicode MS' are added for unicode support
-$font-family-serif:       Garamond, Baskerville, Baskerville Old Face, Hoefler Text, Times New Roman, 'Apple Symbols','Menlo','Arial Unicode MS', serif !default; //'Apple Symbols','Menlo','Arial Unicode MS' are added for unicode support
+$font-family-serif:       Garamond, Baskerville, 'Baskerville Old Face', 'Hoefler Text', 'Times New Roman', Times, 'Apple Symbols','Menlo','Arial Unicode MS', serif !default; //'Apple Symbols','Menlo','Arial Unicode MS' are added for unicode support
```

Comment:
yup. And previously we had some fonts that had spaces and did not have quotes. oops.